openapi/src/parts/workspace.yaml

```diff
@@ -659,6 +659,11 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/WsmPolicyInput'
+        missingAuthDomains:
+          type: array
+          description: The auth domains that the user is missing for this workspace
```

Comment:
Maybe check with @cbookg about what he'd like to call it? In Sam-land it is an auth domain, but in policy-land it is a failed group constraint. I don't feel strongly about what we call it here.

Reply:
chris approved the field in the ticket comment